File: scratch/check_vps_nginx.py
import paramiko

VPS_IP = "195.35.22.13"
VPS_PORT = 22
VPS_USER = "root"
PASSWORD = "SHASTIKARAM#@97may"

def main():
    ssh = paramiko.SSHClient()
    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
    ssh.connect(VPS_IP, port=VPS_PORT, username=VPS_USER, password=PASSWORD, timeout=15)
    
    commands = [
        ("Nginx Config", "cat /etc/nginx/sites-enabled/default || cat /etc/nginx/nginx.conf"),
        ("Var WWW List", "ls -la /var/www"),
        ("Var WWW HTML List", "ls -la /var/www/html || true"),
        ("PM2 list", "pm2 status")
    ]
    
    for title, cmd in commands:
        print(f"\n=== {title} ===")
        stdin, stdout, stderr = ssh.exec_command(cmd)
        print(stdout.read().decode('utf-8'))
        err = stderr.read().decode('utf-8')
        if err:
            print(f"Error: {err}")
            
    ssh.close()

if __name__ == "__main__":
    main()

File: scratch/check_vps_nginx.mjs
import { Client } from 'ssh2';

const conn = new Client();

conn.on('ready', () => {
  console.log('📡 SSH Connection Ready');
  conn.exec('cat /etc/nginx/sites-available/default', (err, stream) => {
    if (err) throw err;
    stream.on('close', (code, signal) => {
      console.log(`Stream closed with code: ${code}`);
      conn.end();
    }).on('data', (data) => {
      console.log('STDOUT:\n' + data);
    }).stderr.on('data', (data) => {
      console.log('STDERR:\n' + data);
    });
  });
}).connect({
  host: '195.35.22.13',
  port: 22,
  username: 'root',
  password: 'SHASTIKARAM@2026'
});
